src/modules/fstab/main.py

```diff
@@ -152,20 +152,22 @@ def generate_crypttab_line_info(self, partition):
         if not mapper_name or not luks_uuid:
             return None
 
-        if mount_point == "/":
+        if mount_point != "/":
```

Comment:
The `/` mount point is specifically excluded from `crypttab` because its unlocking is handled early on. Not sure why you're doing the opposite here?